Formula/mesa.rb

```diff
@@ -93,7 +93,7 @@ def install
         args << "-Dopengl=true"
         args << "-Dgles1=true"
         args << "-Dgles2=true"
-        args << "-Dxvmc=true"
+        args << "-Dgallium-xvmc=true"
```

Comment:
What does this option do?

Reply:
I don't know to be honest, as I am not very deep into the mesa library. However, as mentioned in https://github.com/Homebrew/homebrew-core/issues/88254 I discovered that the current option(name) in the formula is invalid, as it is not listed in the meson build sourcecode, which caused my lokal installation to fail. The "unknown option" error message was resolved. I did some deep dive to find this issue, but as I am no mesa author/contributor/developer I need to confess that fixing this is also a bit of a lucky guess

Reply:
I ask because it seems this option may have been broken for a while (and possibly implicitly disabled) without anyone noticing. If so, we should just deactivate it and remove any associated dependencies, because they're likely not needed.

Reply:
I will try doing so and rerun my local testing scenario to see how the system behaves - will come back with a result as soon as possible

Reply:
See https://bbs.archlinux.org/viewtopic.php?id=261629 and https://www.phoronix.com/scan.php?page=news_item&px=MTU3MDM

```
X-Video Motion Compensation (XvMC) was useful many years ago for video acceleration, but these days it's next to useless for modern video formats and hardware.
```

I think we can safely disable it completely.

Reply:
Cool. Let's do that. We should remove any associated dependencies where we can too.

Reply:
I can do so - however, I just discovered a "new" root casue for the problem which was driving me to open this PR. No matter whether I am running my local build having the option-name fixed to "Dgallium-xvmc" or disabling it completely, the build always fails in a later point of time. I did some further research and it turns out, that the problem is related to the latest version of "meson 0.60". This issue is also already reported at https://gitlab.freedesktop.org/mesa/mesa/-/issues/5576 . However, I manipulated my local Homebrew environment so I was able to use meson 0.59.2 instead of the latest 0.60 version and this way the existing formula works, even with having the 'invalid' Dxvmc option enabled.
So based on these latest findings, could you please help me out making a decision: 

- Should we still remove the option completely as suggested?
- Should I delete this PR to wait for a new mesa version compatible with meson 0.60
- Another idea (like my local manipulation) would be to set the dependency to meson@0.59 - however, as the current meson binary does not support versioning, that would explode the overall topic a bit.

What would you as the experts say?

Reply:
There is a new version of `meson`. Can we try it with that one?

Reply:
Thanks a lot for the hint - tried it on my machine this time with new meson release v0.60.1 and this time it worked when applying the fix to the formula as suggested in this PR (otherwise it fails again as mentioned in the related issue description)

Reply:
So overall question is: apply the fix as suggested here or as stated by @iMichka remove the setting completely

Reply:
I'd say remove it completely

Reply:
Okay will do so - by the way just also tryed to install mesa having this option removed completely and still the original packes having a dependency on mesa (awscli in my case) still works as expected. However, I only reinstalled mesa this time, not other packages, but still I think we are fine.
Going to update the PR